images/installer/root/exports/config.json.template

```diff
@@ -24,7 +24,7 @@
             "PLAYBOOK_FILE=$PLAYBOOK_FILE",
             "ANSIBLE_CONFIG=$ANSIBLE_CONFIG"
         ],
-        "cwd": "/opt/app-root/src/",
+        "cwd": "/usr/share/ansible/openshift-ansible",
```

Comment:
I'm ok with the idea, but I think this may fail. At [this point](https://github.com/sosiouxme/openshift-ansible/blob/8b299f026ceeac66fb7f6565416d860f9abd9f2e/images/installer/root/exports/config.json.template#L105-L112) we are mounting the ``.ssh`` into ``/opt/app-root/src/``. I *think* that line will need to be updated too so it's mounted to the new CWD.

Reply:
I believe for the ssh key it is looking in `$HOME` not `CWD`. I'll test to make sure.

Reply:
Ya, eet works.

Reply:
Sweet!